OPAL/br/src/main/scala/org/opalj/br/Type.scala

```diff
@@ -555,7 +548,7 @@ sealed trait CTIntType extends Type {
  */
 case object CTIntType extends CTIntType {
     final val id = Int.MinValue + 1
-    def toBinaryJavaName: String = throw new UnsupportedOperationException()
+    override def toBinaryJavaName: String = throw new UnsupportedOperationException()
```

Comment:
Why should int-like types not support this getter? They do, just like every other type (except array)
![image](https://github.com/user-attachments/assets/8465cc3e-15e5-4232-b747-b0a7fbf6cb79)

Reply:
They do? They inherit the default implementation toBinaryJavaName = toJava from `Type`. The line you marked is only the companion object that represents all computational int types and thus has no individual binary representation.

Reply:
Damn didnt see that it was the companion object. Sometimes these confuse me when I do not have IDE support...

This is fine then.